pkg/controllers/job/job_controller_handler.go

```diff
@@ -182,8 +191,24 @@ func (cc *Controller) updatePod(oldObj, newObj interface{}) {
 		return
 	}
 
+	// Filter out pods that are not created from volcano job
+	if !IsControlledBy(newPod, helpers.JobKind) {
+		return
+	}
+
+	if newPod.ResourceVersion == oldPod.ResourceVersion {
```

Comment:
Just curious, in which case we will have the same resource version for old and new object?

Reply:
resync